unittest/testshpToTopojson.java

```diff
@@ -0,0 +1,59 @@
+import static org.junit.Assert.*;
+
+import java.io.DataInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+import javax.imageio.ImageIO;
+
+
+import json.converter.csv.CSVReader;
+import json.converter.shp.ShpFileReader;
+import json.geojson.FeatureCollection;
+import json.graphic.Display;
+import json.tools.Compress;
+import json.tools.Toolbox;
+import json.topojson.algorithm.ArcMap;
+import json.topojson.api.TopojsonApi;
+import json.topojson.geom.sub.Entity;
+import json.topojson.topology.Topology;
+
+import org.geotools.referencing.CRS;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import com.google.gson.Gson;
+import org.opengis.referencing.FactoryException;
+
+//import org.opengis.referencing.NoSuchAuthorityCodeException;
+import org.opengis.referencing.crs.CoordinateReferenceSystem;
+
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+
+
+public class testshpToTopojson {
+
+	@Test
+	public void test() throws IOException, FactoryException  {
+	
+		// Your text data (assuming it's a CRS specification in a specific encoding)
+        String crsSpec = "EPSG:4326";  // Replace with your actual CRS specification
+
+        // Decode the text using UTF-8 encoding (or the appropriate encoding)
+        byte[] crsBytes = crsSpec.getBytes(StandardCharsets.UTF_8);
+        String decodedCRS = new String(crsBytes, StandardCharsets.UTF_8);
+		
+				TopojsonApi.shpToTopojsonFile("./data/example.shp", CRS.decode(decodedCRS),
+						"./web/topojson_hale.json", 
```

Comment:
If that's the target file the data is written to I would expect the following things:

- use a temporary directory for the file and clean up after the test is done (and don't add it to the Git repository like the current file)
- read the result and verify what we want to test (that null values are represented as expected?)

Reply:
I have moved the small method that was actually there just to use one of our .shp file produced with hale studio.
I have kept the example for future use.
The null values are changed [here in HS](https://github.com/halestudio/hale/pull/1082/commits/b62b783f25124f28d40a69c392586ef5375c4006)